.gitignore

```diff
@@ -8,4 +9,5 @@ cross
 *.swp
 .idea
 *.iml
+*.test
```

Comment:
Yep, for this workspace I think

On Friday, July 8, 2016, Riyaz Faizullabhoy <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/docker/notary/pull/808#discussion_r70102210>:
>
> > @@ -8,4 +9,5 @@ cross
> >  *.swp
> >  .idea
> >  *.iml
> > +*.test
>
> I'm good with ignoring this file extension, but out of curiosity what is
> it used for? VSCode metadata?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/notary/pull/808/files/1ff0f8c416206fbd7f9bf80bda0c3a7c516a6fa1#r70102210>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANn4PQdmoXN7_9X4sCTr9RvCb889fxFks5qTnsCgaJpZM4JAytD>
> .
>